src/main/java/edu/kis/powp/jobs2d/commands/OperateToCommand.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.commands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+    private int x;
+    private int y;
+    private final Job2dDriver driver;
+
+    OperateToCommand(Job2dDriver dr) {
+        this.driver = dr;
+    }
+
+    @Override
+    public void execute() {
+        driver.operateTo(x, y);
+    }
+
+    public int getX() {
+        return x;
+    }
+
+    public OperateToCommand setX(int x) {
```

Comment:
I don't think that fluent syntax, in this case, is justified. Setting X and Y in the constructor is enough.